### PR TITLE
[UITest] Add NUnit Timeout attribute to UI Tests

### PR DESCRIPTION
**Pull request by kdubau**

### CODE REVIEW

File: main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs [hunk 1]
```diff
@@ -30,11 +30,11 @@
 
 namespace UserInterfaceTests
 {
-	[TestFixture]
+	[TestFixture, Timeout(60000)]
 	[Category ("Git")]
 	public class GitTests : VCSBase
 	{
-		[Test]
+		[Test, Timeout(90000)]
```
Comment (manish) [inline main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs]: TestGitClone is 90s and GitTests is 60s.

Comment (kdubau) [inline main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs]: It's intentional. The clone test takes a bit longer than the others.

Comment (manish) [inline main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs]: but time taken by a fixture includes all the tests.

So even if GitClone passes (say 70s), GitTests will fail as it took more than 60s

Comment (kdubau) [inline main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs]: The timeout on TestFixture is just the default for any subordinate test cases. According to the [docs] (http://www.nunit.org/index.php?p=timeout&r=2.5).
> The attribute may also be specified on a fixture or assembly, in which case it indicates the default timeout for any subordinate test cases.

Comment (manish) [inline main/tests/UserInterfaceTests/VersionControlTests/Git/GitTests.cs]: :+1:

**Pull request merged**